EDITING.md

```diff
@@ -119,16 +119,16 @@ Here are some tips on how to maintain the parallel language structure in markup.
     <span class="codepoint" translate="no"><span lang="ja">。</span> [<span class="uname">U+3002 IDEOGRAPHIC FULL STOP</span>]</span>
     ```
 
-- To link to a section, use the Respec feature, ie. link to the id on the section tag and leave the link text empty.
+- To link to a section, use the ReSpec feature, ie. link to the id on the `section` tag and leave the link text empty.
```

Comment:
>  and leave the link text empty

I don't think that text is needed any more.  Same applies to the figure instructions.

Reply:
Thanks. Updated now.